framework/src/Volo.Abp.AspNetCore.Mvc/Volo/Abp/AspNetCore/Mvc/Json/MvcCoreBuilderExtensions.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Buffers;
+using System.Text.Json;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Formatters;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Options;
+using System.Text.Encodings.Web;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.ObjectPool;
+
+namespace Volo.Abp.AspNetCore.Mvc.Json
+{
+    public static class MvcCoreBuilderExtensions
+    {
+        public static IMvcCoreBuilder AddAbpHybridJson(this IMvcCoreBuilder builder)
+        {
+            //SystemTextJsonInputFormatter
+            builder.Services.AddTransient(provider =>
+            {
+                var jsonOptions = provider.GetRequiredService<IOptions<JsonOptions>>();
+                var logger = provider.GetRequiredService<ILoggerFactory>().CreateLogger<SystemTextJsonInputFormatter>();
+                return new SystemTextJsonInputFormatter(jsonOptions.Value, logger);
+            });
+
+            builder.Services.AddTransient<DefaultObjectPoolProvider>();
```

Comment:
TryAddTransient would be better.

Reply:
https://github.com/abpframework/abp/pull/6054